.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: Build Eigenpy
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
I forgot the `devel` branch. You can enable both.

Reply:
You're right. I will do.